processing/src/main/java/org/apache/druid/query/AbstractQueryLogic.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.query;
+
+import com.google.inject.Inject;
+
+/**
+ * Executes the query by utilizing the given walker.
+ */
+public abstract class AbstractQueryLogic implements QueryLogic
+{
+  protected QueryRunnerFactoryConglomerate conglomerate;
+
+  @Inject
+  public void initialize(QueryRunnerFactoryConglomerate conglomerate)
+  {
+    this.conglomerate = conglomerate;
+  }
+}
```

Comment:
I'm not sure how much value this abstract class is really adding anymore, but beyond this comment, changes here don't need to block anything.

Reply:
good idea - removed it!